### PR TITLE
Update serving doc

### PR DESCRIPTION
**Pull request by Romanticoseu**
## Description

Update serving doc

### CODE REVIEW

File: python/llm/src/bigdl/llm/serving/README.md [hunk 1]
```diff
@@ -1,21 +1,24 @@
...
 
 - [Install](#install)
-- [Models](#models)
-- [Boot Service](#start-the-service)
-  - [Web GUI](#serving-with-webgui)
-  - [RESTful API](#serving-with-openai-compatible-restful-apis)
+- [Start the service](#start-the-service)
+  - [Launch controller](#launch-controller)
+  - [Launch model worker(s) and load models](#launch-model-workers-and-load-models)
+    - [Bigdl model worker](#bigdl-model-worker)
```
Comment (glorysdj) [inline python/llm/src/bigdl/llm/serving/README.md:14]: Bigdl -> BigDL
vllm -> BigDL vLLM

File: python/llm/src/bigdl/llm/serving/README.md [hunk 2]
```diff
@@ -27,80 +30,84 @@ pip install --pre --upgrade bigdl-llm[all]
...
+#### Bigdl model worker
 
-##### Serving with WebGUI
-
-To serve using the Web UI, you need three main components: web servers that interface with users, model workers that host one or more models, and a controller to coordinate the web server and model workers.
-
-###### Launch the Controller
 ```bash
-python3 -m fastchat.serve.controller
+# In CPU
```
Comment (glorysdj) [inline python/llm/src/bigdl/llm/serving/README.md:68]: On CPU

File: python/llm/src/bigdl/llm/serving/README.md [hunk 3]
```diff
@@ -27,80 +30,84 @@ pip install --pre --upgrade bigdl-llm[all]
...
+python3 -m bigdl.llm.serving.model_worker --model-path PATH/TO/bigdl-7b --device cpu
+
+# In XPU
```
Comment (glorysdj) [inline python/llm/src/bigdl/llm/serving/README.md:71]: On GPU

File: python/llm/src/bigdl/llm/serving/README.md [hunk 4]
```diff
@@ -27,80 +30,84 @@ pip install --pre --upgrade bigdl-llm[all]
...
+#### BigDL vLLM model worker
 
 We also provide the `vllm_worker` which uses the [vLLM](https://github.com/intel-analytics/BigDL/tree/main/python/llm/example/CPU/vLLM-Serving) engine for better hardware utilization.
 
-To run using the `vllm_worker`, just simply uses the following command:
+To run using the `vLLM_worker`, just simply uses the following command:
+
 ```bash
-python3 -m bigdl.llm.serving.vllm_worker --model-path meta-llama/Llama-2-7b-chat-hf --device cpu/xpu # based on your device
+# On CPU
```
Comment (gc-fu) [inline python/llm/src/bigdl/llm/serving/README.md:90]: `vllm_worker` does not rely on changing the model name.

**Review by glorysdj — APPROVED** [review]
LGTM

**Pull request merged**